texastribune/styleguides
# Move jekyll functionality from makefile to grunt

To unify the interface and make it so you need fewer windows open, this PR moves the jekyll functionality from the makefile shortcut to grunt. I tried this using `grunt-jekyll` but it sucked. Based on a conversation at ATXSass, I tried again using a shell task.
#### Testing
1. `npm install` to update requirements
2. `grunt dev` to start watcher and server
3. open the jekyll site in your browser, probably http://localhost:4000/
4. make a trivial css, change. I added `background: mediumpurple !important;` to the html
5. check to see that everything is purple
6. make a trivial html change. I added `moo` to the beginning of `_includes/nav.html`
7. check to see that you're mooing
#### Next?

there's a grunt task for automagically opening the browser too, but I think that's going too far.
#### Smells

There's some code I thought I might need/want but I don't think we'll ever use it. See the "`jekyllWatch` is not used, but here for reference" comment
